scripts/setup_fullnode.sh

```diff
@@ -124,15 +124,67 @@ function preDockerCompose() {
 
     export IOTEX_HOME IOTEX_MONITOR_HOME IOTEX_IMAGE
 
-    curl -Ss https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/monitor/docker-compose.yml.gateway > $IOTEX_MONITOR_HOME/docker-compose.yml.gateway
-    curl -Ss https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/monitor/docker-compose.yml > $IOTEX_MONITOR_HOME/docker-compose.yml.default
-    curl -Ss https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/monitor/.env > $IOTEX_MONITOR_HOME/.env
+    curl -Ss --connect-timeout 10 \
```

Comment:
we can build a shell function like
```
function getNetResource(){
}
```
`do http code judge` and `retry` can do in the func

Reply:
maybe we can parse curl return information, and do retry when got 404 http code. 
during retrying, print some log, and show the number of retries to user maybe user friendliness

Reply:
@huangzhiran Implemented new function for download. Can you review? If retry occurs then user is notified about retry.